src/routes/safe/components/Apps/hooks/useAppList.ts

```diff
@@ -89,8 +103,9 @@ const useAppList = (): UseAppListReturnType => {
 
   return {
     appList,
-    staticAppsList,
+    apiAppsList,
```

Comment:
Why do you need to return both appList and apiAppsList? I think it can be merged into one

Reply:
Agree. You could add a new property into the `SafeApp` object inside the `appList` array